plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/InviteCommand.java

```diff
@@ -68,4 +117,41 @@ public void execute(@CanManage User deputy, Guild guild, String[] args) {
         invitedUser.sendMessage(formatter.format(this.messages.inviteToInvited));
     }
 
+    public static final class InviteCommandCompleter implements Completer {
```

Comment:
Wyrzuc to do oddzielnej klasy (tam gdzie inne completery) i zrób package-private

Reply:
Ale technologia. Mamy taką klasę? 😳

Reply:
Peridot szczerze to tutaj zrobiłbym package `completer` i tam wszystko to wrzucił

Reply:
Obecnie jest wszystko w `net.dzikoysk.funnyguilds.command`

Reply:
To nie tak działa, takie rzeczy zgodnie ze sztuką powinny byc package-private - nie bez powodu wszystkie completery nie bez powodu są aktualnie package-private